src/public/index.html

```diff
@@ -8,8 +8,11 @@
     <title>App</title>
 
     <base href="/">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/bootstrap/3.3.7/css/bootstrap.min.css">
   </head>
   <body>
+    <div nav-bar></div>
     <div ui-view class="site"></div>
+    <script src="https://cdn.jsdelivr.net/g/jquery@3.1.1,tether@1.4.0,bootstrap@3.3.7"></script>
   </body>
 </html>
```

Comment:
Ended up needed jquery first but could remove teather as we are using bootstrap 3 again.